db/re/item_db_etc.yml

```diff
@@ -11609,11 +11609,9 @@ Body:
       BuyingStore: true
       DropEffect: CLIENT
     Script: |
+      bonus bCritical,9;
       if (BaseLevel<100) {
-         bonus bCritical,9;
-      }
-      else {
-         bonus bCritical,10;
+         bonus bCritical,1;
```

Comment:
For me the previous item script was fine
```
CRI + 9.
At base level 100 or higher, CRI + 1 added, increases critical damage by 5%. 
```

Reply:
Thanks